### PR TITLE
Support synchronous check for whether the message was published

### PR DESCRIPTION
**Pull request by kibertoad**
This is primarily useful for checking that message _wasn't_ published.

### CODE REVIEW

File: packages/sns/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "prepublishOnly": "npm run build:release"
   },
   "dependencies": {
-    "@lokalise/node-core": "^10.0.0",
+    "@lokalise/node-core": "^10.0.1",
```
Comment (CarlosGamero) [inline packages/sns/package.json:28]: Should we update it also on AMQP and SQS?

**Review by CarlosGamero — APPROVED** [review]

File: packages/sns/package.json (hunk 1 above)
Comment (kibertoad) [reply packages/sns/package.json:28]: they are already up-to-date, as far as I can see

Comment (CarlosGamero) [reply packages/sns/package.json:28]: Oh, sorry 🙇, thank youu 🙏

Comment (kibertoad) [reply packages/sns/package.json:28]: np!

**Pull request merged**